target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -75,6 +77,15 @@ ramips_setup_macs()
 	local label_mac=""
 
 	case $board in
+	ampedwireless,ally-00x19k)
+		lan_mac=$(mtd_get_mac_ascii hwconfig HW.LAN.MAC.Address)
```

Comment:
Either you set the correct address for gmac in DTS or use lan_mac here. DTS is preferred.
But please don't do both.

Same goes for label-mac-device vs. label_mac here.

Reply:
ok thanks